packages/cli/src/commands/setup/ui/chakra-ui.js

```diff
@@ -0,0 +1,77 @@
+import execa from 'execa'
+import Listr from 'listr'
+
+import c from '../../../lib/colors'
+
+import {
+  checkStorybookStatus,
+  configureStorybook,
+} from './tasks/configure-storybook'
+import { checkSetupStatus, wrapWithChakraProvider } from './tasks/setup-chakra'
+
+export const command = 'chakra-ui'
+export const description = 'Set up Chakra UI'
+
+export function builder(yargs) {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+  yargs.option('install', {
+    alias: 'i',
+    default: true,
+    description: 'Install packages',
+    type: 'boolean',
+  })
+}
+
+export async function handler({ force, install }) {
+  const packages = [
+    '@chakra-ui/react',
+    '@emotion/react@^11',
+    '@emotion/styled@^11',
+    'framer-motion@^4',
+  ]
+
+  const tasks = new Listr([
+    {
+      title: 'Installing packages...',
+      skip: () => !install,
+      task: () => {
+        return new Listr([
+          {
+            title: `Install ${packages.join(', ')}`,
+            task: async () => {
+              await execa('yarn', [
+                'workspace',
+                'web',
+                'add',
+                '-D',
```

Comment:
@TimKolberger or @thedavidprice can you explain why we're adding these packages to the dev dependencies? Neither chakra-ui's documentation, nor Mantine's (which I'm working on now) suggest we should install to dev dependencies. Is this a Redwood thing, an oversight, or something else? 

I have no reason to think Mantine's components should be dev-only, but I don't want to deviate from the existing setup behavior from Chakra, since this seems meaningful. Thoughts?

Reply:
You can see my progress here: https://github.com/redwoodjs/redwood/pull/4900/files

Reply:
@nickpdemarco devDeps is fine because there's a build step with Webpack, and the Build Env is set to development. So I think this has become a default as a preference/practice. We also use devDep for Tailwind setup packages.

That said, you could make your own argument otherwise. Thanks for asking!